lib/keyboard_actions.dart

```diff
@@ -417,9 +409,7 @@ class KeyboardActionstate extends State<KeyboardActions>
         ? _kBarSize
         : 0; // offset for the actions bar
 
-    final keyboardHeight = EdgeInsets.fromWindowPadding(
-            WidgetsBinding.instance.window.viewInsets,
-            WidgetsBinding.instance.window.devicePixelRatio)
+    final keyboardHeight = MediaQuery.viewInsetsOf(context)
         .bottom;
```

Comment:
The problem was here. `keyboardHeight` was wrong, causing `_offset` to be bigger than it should